src/Library/demos/Web View/main.js

```diff
@@ -0,0 +1,91 @@
+import WebKit from "gi://WebKit?version=6.0";
+const GObject = imports.gi.GObject;
+const { Uri } = imports.gi.GLib;
+
+const container = workbench.builder.get_object("container");
+const button_back = workbench.builder.get_object("button_back");
+const button_forward = workbench.builder.get_object("button_forward");
+const button_reload = workbench.builder.get_object("button_reload");
+const button_stop = workbench.builder.get_object("button_stop");
+const url_bar = workbench.builder.get_object("url_bar");
+const web_view = new WebKit.WebView({
+  zoom_level: 0.8,
```

Comment:
any particular reason?

Reply:
Oh it was because you couldn't really see much of the webpage because the view is small, so I decreased the zoom level. It also gave me a reason to show the property.

Reply:
Interesting find. I can confirm it's the same with `Box` and `append`.